src/cli/configure/analysis/__snapshots__/project.test.ts.snap

```diff
@@ -64,11 +70,15 @@ node_modules*/
 npm-debug.log
 package-lock.json
 yarn-error.log
+# end managed by skuba
+
+
```

Comment:
This doesn't seem right; do we need to trim excess newlines?

Reply:
I think this is from `.gitignore` having an extra new line in the template